dspace-api/src/main/java/org/dspace/disseminate/CitationDocumentServiceImpl.java

```diff
@@ -280,7 +280,24 @@ public File makeCitedDocument(Context context, Bitstream bitstream)
         try {
             Item item = (Item) bitstreamService.getParentObject(context, bitstream);
             sourceDocument = sourceDocument.load(bitstreamService.retrieve(context, bitstream));
-            PDPage coverPage = new PDPage(PDRectangle.LETTER); // TODO: needs to be configurable
+	    // Page format can be either LETTER (default) or A4
+	    String pageformat_cfg = configurationService.getProperty("citation-page.page_format");
+	    PDRectangle pageformat = PDRectangle.LETTER;
+	    if (pageformat_cfg == null) {
+		pageformat = PDRectangle.LETTER;
+	    } else {
+		String pf = pageformat_cfg.toUpperCase();
+		if (pf.equals("A4")) {
+		    pageformat = PDRectangle.A4;
+		} else if (pf.equals("LETTER")) {
+		    pageformat = PDRectangle.LETTER;
+		} else {
+		    log.info("Citation-page: Unknown page format ' " + pageformat_cfg + "', using LETTER.");
+		    pageformat = PDRectangle.LETTER;
+		}
```

Comment:
Just some minor feedback on this: this if loop could probably be simplified a bit.. since pageformat is instantiated as PDRectangle.LETTER already, the test on pageformat_cfg could be for != null, and then an else block wouldn't be needed.. the final else on the inside test is probably fine as an explicit default and is a good opportunity to log that info message. Also, not an official style rule but DSpace variables tend to be in camelCase, eg pageformatConfig, rathter than pageformat_cfg.
Just feedback and not a blocker, I'm still +1 on the PR anyway :)